.github/CODEOWNERS

```diff
@@ -0,0 +1,194 @@
+
+/tutorials/3d/                                      @godotengine/documentation @godotengine/3d-nodes
+
+# TODO: Should any of these also have rendering ownership when rendering related?
+
+## --------------- Animation -----------------------
+
+/tutorials/animation/                               @godotengine/documentation @godotengine/animation
+
+## --------------- Assets Pipeline -----------------
+
+/tutorials/assets_pipeline/                         @godotengine/documentation @godotengine/import
+
+# TODO: Any additional ownership for 3D and translations?
+
+## --------------- Audio ---------------------------
+
+/tutorials/audio/                                   @godotengine/documentation @godotengine/audio
+
+## --------------- Best Practices ------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Editor --------------------------
+
+/tutorials/editor/                                  @godotengine/documentation @godotengine/docks
+/tutorials/editor/*script*                          @godotengine/documentation @godotengine/script-editor
+
+# TODO: Who should control this beyond docs?
+
+## --------------- Export --------------------------
+
+/tutorials/export/*android*                         @godotengine/documentation @godotengine/android
+/tutorials/export/*ios*                             @godotengine/documentation @godotengine/ios
+/tutorials/export/*linux*                           @godotengine/documentation @godotengine/linux-bsd
+/tutorials/export/*macos*                           @godotengine/documentation @godotengine/macos
+/tutorials/export/*visionos*                        @godotengine/documentation @godotengine/ios
+/tutorials/export/*web*                             @godotengine/documentation @godotengine/web
+/tutorials/export/*windows*                         @godotengine/documentation @godotengine/windows
+
+# TODO
+
+## --------------- i18n ----------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Inputs --------------------------
+
+/tutorials/inputs/                                  @godotengine/documentation @godotengine/input
+
+## --------------- IO ------------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Math ----------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Migrating -----------------------
+
+/tutorials/migrating/                               @godotengine/documentation
+
+## --------------- Navigation ----------------------
+
+/tutorials/navigation/                              @godotengine/documentation @godotengine/navigation
+
+## --------------- Networking ----------------------
+
+/tutorials/networking/                              @godotengine/documentation @godotengine/network
+
+## --------------- Performance ---------------------
+
+# TODO: Who should control this beyond documentation?
```

Comment:
I'd say CPU optimization is best reviewed by core, and gpu optimization is best reviewed by rendering.